components/Home/index.tsx

```diff
@@ -34,14 +35,18 @@ const VestingBox = styled(BalanceBox)`
 `
 
 const MsigHome = () => {
-  const { AvailableBalance, Balance } = useMsig()
+  const { AvailableBalance, Balance, loading } = useMsig()
   return (
     <>
       <BalanceBox>
         <h3>Available Balance</h3>
-        <span className='value'>
-          {makeFriendlyBalance(AvailableBalance, 6, true)}
-        </span>
+        {loading ? (
+          <LoadingIcon size='48px' />
```

Comment:
The default size is `3em` which should be the same as `48px` here?
Otherwise it's best to express in `em` as it scales with when the user changes their root font-size with the browser.